packages/documentation/copy/en/reference/Decorators.md

```diff
@@ -6,17 +6,20 @@ oneline: TypeScript Decorators overview
 translatable: true
 ---
 
-## Introduction
-
-<blockquote class='bg-reading'>
-  <p>Further Reading:<br /><a href='https://saul-mirone.github.io/a-complete-guide-to-typescript-decorator/'>A Complete Guide to TypeScript Decorators</a></p>
+<blockquote>  
```

Comment:
Does this need to be `<blockquote>`> Can it just be `>` like all of the other ones?

Note that this one:

![image](https://github.com/microsoft/TypeScript-Website/assets/5341706/d88250b9-3836-4039-8798-9d2bd9c654bd)

looks "off" compared to the others in the page:

![image](https://github.com/microsoft/TypeScript-Website/assets/5341706/a7554e80-4f59-4fd3-a01b-3394cf27779b)

Reply:
the difference in 'NOTE' spacing was there in the original, in any case, normalizing this.